lab-andrew/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "lab-andrew",
+  "version": "0.1.0",
+  "description": "cowsay server req/res",
+  "main": "server.js",
+  "scripts": {
+    "test": "gulp-lint"
+  },
+  "author": "andrewpeterwatson@gmail.com",
+  "license": "ISC",
+  "devDependencies": {
+    "gulp": "^3.9.1",
+    "gulp-lint": "^2.0.0"
```

Comment:
This should be 'gulp-eslint', make sure to use npm to add your dependencies to your package.json to avoid typos.